cmd/test.go

```diff
@@ -293,9 +293,12 @@ func runTests(ctx context.Context, txn storage.Transaction, runner *tester.Runne
 	go func() {
 		defer close(dup)
 		for tr := range ch {
-			if !tr.Pass() {
+			if tr.Fail {
 				exitCode = 2
 			}
+			if exitCode == 0 && tr.Skip {
+				exitCode = 3
```

Comment:
I understand the first change, but not this one. Should a skipped test still count as a failure wrt exit code? 🤔

Reply:
From the [comment](https://github.com/open-policy-agent/opa/issues/3773#issuecomment-910831199) made by @tsandall yes.

Reply:
Thanks Kirk! That would probably mean failing CI/CD jobs due to ignored tests, which kinda defeats the point of skipping them in the first place.. so yeah. I hope we can reconsider this.

Reply:
I am ok considering a skipped test as exit code 0.  Perhaps @tsandall can chime in.  I had assumed that I would just check if the exit code == 3 vs 2 in cicd and do something like

opa test 
if [[ $? -eq 0 ]] || [[ $? -eq 3 ]]; then
  # pass
else
  # fail
fi

Reply:
That’s an option, though having to add special treatment for ignored tests (or even having to know about the different exit codes) would be annoying, and I can’t remember ever having to do that for any other test framework.